client/src/main/java/com/alibaba/nacos/client/security/SecurityProxy.java

```diff
@@ -42,45 +42,45 @@
  * @since 1.2.0
  */
 public class SecurityProxy {
-    
+
```

Comment:
Please do not change the indent.
Please use nacos' code style to reformat codes.